index.js

```diff
@@ -37,7 +37,8 @@ const defaultChannels = [
   'code',
   'confessions',
   'counttoamillion',
-  'hack-night',
+  'sprig',
+  'community',
```

Comment:
I think #hack-night should still be a default channel. It could be a way for new people to engage with the community, live, for a weekly event, on Zoom.

```suggestion
  'community',
  'hack-night',
```